### DISCUSSION TITLE
VAE vs autoencoder

### DISCUSSION THREAD

**Discussion by axu-git** [clementchadebec/benchmark_VAE#136] [General]
If I run a RHVAE with encoder = decoder = none, it seems from the documentation that encoder = decoder = MLP. In this case, is it true that this is just an autoencoder (and not a VAE) because there is no distribution?

Thanks!

**Comment by clementchadebec** [comment]
No, it is still a VAE since the MLPs will provide in this case the parameters of the distributions. Mean and covariance of the posterior q(z|x) are given by the encoder while the mean of p(x|z) is given by the decoder.

**Comment by axu-git** [comment]
So we still have q(z | x) and p(x | z) are each Gaussians even if encoder = decoder = none?

**Reply by clementchadebec** [reply]
yes, this is correct

**Comment by axu-git** [comment]
Great thanks!
